CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ project(onnx C CXX)
 option(ONNX_BUILD_BENCHMARKS "Build ONNX micro-benchmarks" OFF)
 
 option(BUILD_ONNX_PYTHON "Build Python binaries" OFF)
-option(ONNX_NO_WERROR "Disable use of Werror" OFF)
+option(ONNX_WERROR "Build with Werror" OFF)
```

Comment:
Would better be: "Treat warning as error"

I hope this option also works for Windows.